### DISCUSSION TITLE
Dependencies not enabled for requirements.txt file

### DISCUSSION THREAD

**Discussion by mvanchaa** [community/community#35469] [Pull Requests]
Text file is considered as a binary file and does not show up as a dependency in a pull request. Attaching the screenshot

![image](https://user-images.githubusercontent.com/99682944/194626018-297b1f34-d1c5-4b26-8dc6-c9f0567a9167.png)

**Comment by reillysiemens** [comment]
For a little extra context, our team discovered this as part of https://github.com/AzureAD/microsoft-authentication-cli/pull/157/. Clicking the triple dot menu and selecting `View file` makes the contents visible, but it doesn't seem like we should have to jump through that hoop to review a dependency addition.
